### DISCUSSION TITLE
Admin Perms

### DISCUSSION THREAD

**Discussion by KikoPT1234** [pocketbase/pocketbase#3784] [Q&A]
I've noticed that, although PocketBase allows you to create multiple admin accounts, there really isn't a use for them because they all have equal rights in the Admin UI. I wanted to know if there's any way/workaround to add permissions to admins, so that one account can only access X collections in the Admin UI, while the other account has access to other collections. In my opinion, this provides more use for the admin accounts.
Although we could make a normal auth collection, this would then mean creating a "staff" dashboard _in addition_ to the main app, which sounds a bit unproductive to me considering that the Admin UI offers everything we need in terms of editing.

**Comment by erickythierry** [comment]
I would also like to know if this is possible

**Reply by ganigeorgiev** [reply]
It is not supported at the moment. It was suggested before (eg. https://github.com/pocketbase/pocketbase/discussions/94#discussioncomment-3725219) but for now remains out of the scope for v1 at least until we stabilize the exposed Go APIs and prepare a plugin interface for the Admin UI. All of this will be handled somewhere after the Presentator v3 release (see https://github.com/pocketbase/pocketbase/discussions/3271).
